test_files/declare.sickle.ts

```diff
@@ -1,5 +1,17 @@
-declare module DeclareTest {
+declare module DeclareTestModule {
   export class Foo {
     constructor();
   }
 }
+
+declare interface DeclareTestInterface {
```

Comment:
`@record` on an `@interface` should make it structurally typed, `@struct` is a good idea, too.

If this works, it's fine. I remember some discussion about how Closure only does structural optimization if it finds code that flows into an `@record` annotated type. We currently don't do anything like that (no type annotations in the output code, just `?`), so let's not go there.